xml/net_samba.xml

```diff
@@ -1369,24 +1382,52 @@ No shadow copies found in system.</screen>
  <sect1 xml:id="sec.samba.info">
   <title>For More Information</title>
 
-  <para>
-   Documentation for Samba ships with the <systemitem>samba-doc</systemitem>
-   package which is not installed by default. Install it with <command>zypper
-   install samba-doc</command>. Enter <command>apropos</command>
-   <option>samba</option> at the command line to display some manual pages or
-   browse the <filename>/usr/share/doc/packages/samba</filename> directory for
-   more online documentation and examples. Find a commented example
-   configuration (<filename>smb.conf.SUSE</filename>) in the
-   <filename>examples</filename> subdirectory. Another file to look for Samba
-   related information is
-   <filename>/usr/share/doc/packages/samba/README.SUSE</filename>.
-  </para>
-
-  <para>
-   The Samba HOWTO (see <link xlink:href="https://wiki.samba.org"/>) provided
-   by the Samba team includes a section about troubleshooting. In addition to
-   that, Part V of the document provides a step-by-step guide to checking your
-   configuration.
-  </para>
+  <itemizedlist>
+   <listitem>
+    <formalpara>
+     <title>Man Pages:</title>
+     <para>
+      To see a list of all man pages installed with the package
+      <package>samba</package>, run <command>apropos samba</command>.
+      You can then open any of the man pages with
+      <command>man <replaceable>NAME_OF_MAN_PAGE</replaceable></command>.
+     </para>
+    </formalpara>
+   </listitem>
+   <listitem>
+    <formalpara>
+     <title>&suse;-specific README file:</title>
+     <para>
+      The package <package>samba-client</package> contains the file
+      <filename>/usr/share/doc/packages/samba/README.SUSE</filename>.
+     </para>
+    </formalpara>
+   </listitem>
+   <listitem>
+    <formalpara>
+     <title>Additional Packaged Documentation:</title>
+     <para>
+      Install the package <systemitem>samba-doc</systemitem> with
+      <command>zypper install samba-doc</command>.
+     </para>
+    </formalpara>
+    <para>
+     This documentation installs into
+     <filename>/usr/share/doc/packages/samba</filename> and contains an HTML
```

Comment:
split into 2 sentences: 'It consists of an HTML version of the man pages and a library of [....]'